src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -452,27 +475,16 @@
      * @param <T> expected inventory type, specified as generic to allow easy
      *      pseudo-duck-typing
      * @return the query result
-     */
-    <T extends Inventory> T query(String... names);
```

Comment:
That would be a diff bug. There was a `Translation` method originally; it was posted just above this method, but the diff thinks the original method was part of the `InventoryProperty` entry.

I removed the `String` version both because it was never implemented and because it was inconsistently referred to: the API referred to it as names, but the implementation referred to it as expressions.